docs/content/troubleshooting.md

```diff
@@ -59,56 +59,74 @@ The most common causes of this exception are `bool` and `System.Boolean`, both o
+  * Ensure that the GameObject containing your MonoBehaviour is associated with a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity). You can verify this by examining whether a [SpatialOS component]({{urlRoot}}/content/glossary#spatialos-component) MonoBehaviour is added to your GameObject at [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime).
+   [comment]: <> (TODO: make this clearer - what is meant by "SpatialOS component" here - do we mean a GameObject component called SpatialOS component? And how do we check whether this is added to the MonoBehaviour? . JIRA:https://improbableio.atlassian.net/browse/UTY-1575 and https://improbableio.atlassian.net/browse/TC-169.)
+  * Ensure that you only access Requirables in a MonoBehaviour while the MonoBehaviour is enabled. Requirables are supposed to be null while the MonoBehaviour is disabled. Note that certain Unity event methods like `OnTriggerEnter` or `OnCollisionEnter` may be invoked, even if a MonoBehaviour is disabled.
+  [comment]: <> (TODO: make this fix clearer - what is meant by "Requireables" - we haven't defined this. JIRA: https://improbableio.atlassian.net/browse/UTY-1575 and https://improbableio.atlassian.net/browse/TC-169.)
 
-#### A MonoBehaviour containing fields using the Require attribute remains disabled when I expect it to be enabled
+<br/>
+#### <li> <b> A MonoBehaviour containing fields using the `Require` attribute remains disabled when I expect it to be enabled </b>
 
 **Causes and fixes**<br/>
 
 This can be caused by multiple problems. Follow the checklist below to discover the cause and fix:
 
-  * Ensure that `GameObjectRepresentationHelper.AddSystems` is called when initializing your [`Worker`]({{urlRoot}}/content/glossary#worker).
-  * Ensure that the GameObject containing your MonoBehaviour is associated with a SpatialOS entity. You can verify this be examining whether a `SpatialOSComponent` MonoBehaviour was added to your GameObject at runtime.
-  * Ensure that the [requirements]({{urlRoot}}/content/gameobject/interact-spatialos-monobehaviours) for all your fields in your MonoBehaviour are satisfied. You can verify this by examining the SpatialOS entity associated with your GameObject in the [Inspector]({{urlRoot}}/content/glossary#inspector). In the Inspector, ensure that all relevant components are attached to your SpatialOS entity and that read and write access is delegated correctly for your worker.
+  * Ensure that `GameObjectRepresentationHelper.AddSystems` is called when initializing your [worker]({{urlRoot}}/content/glossary#worker).
+  * Ensure that the GameObject containing your MonoBehaviour is associated with a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity). You can verify this be examining whether a `SpatialOSComponent` MonoBehaviour was added to your GameObject at [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime).
```

Comment:
I don't think this should link to "SpatialOS Runtime", since "runtime" here is being used in the sense of "while running".

Reply:
Mmm - I see. Think we should use a different way to say this completely.

Reply:
"   * Ensure that the GameObject containing your MonoBehaviour is associated with a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity). You can verify this by examining whether a `SpatialOSComponent` MonoBehaviour was added to your GameObject when you run your game.

Reply:
@anne-edwards Would you make the suggested change so I can add it? I can't see to add my own edits.